tests/yaml/pip988.yml

```diff
@@ -0,0 +1,38 @@
+# https://github.com/pypa/pip/issues/988
+# see comment from benoit-pierre
```

Comment:
suggestion: we should add a direct link to benoit-pierre's comment, since the issue has over 100 comments and many many examples.

Maybe, this file should also be renamed, since 988 is the overall tracking issue for the resolver and has a lot more failure cases that are mentioned directly / get linked into it via GitHub's cross-linking.

Reply:
Yes, I updated the link in the comment.  However, what would you rename the file to?

Reply:
Something about "state" -- since we're testing how already-installed packages affect the resolution?